scripts/docker/entrypoint_ci.sh

```diff
@@ -436,6 +440,14 @@ else
                 echo "${COLOR_YELLOW}Skip ${providers_dir} as the directory does not exist.${COLOR_RESET}"
             fi
         done
+    elif [[ ${TEST_TYPE} =~ PlainAsserts ]]; then
+        # Those tests fail when --asert=rewrite is set, therefore we run them separately
```

Comment:
Can you maybe also include a link to one or more of the issues/PRs which contain the greater context for this workaround?

Reply:
Yeah. I created them later, but sure. I can add them :) . And yeah. I absolutely don't love the workarounds either. It took me - unnecessarily at least couple of good hours and even more hair pulled - this time from the side of my hair (I seem t to have even less of them now). I actually had to do bisect on the REGEX pattern (adding letter by letter) in python_files to be able to find which files are causing the problems.  

But yeah. I've learned quite a bit - so it is not entirely lost effort (and I **just** signed a CLA with datastax as they would not accept my "add one line of comment" contribution :D...

Reply:
I've added the links. Indeed - I have forgotten to add them in the sources after I created issues/PRs